src/TestSuite/Constraint/Response/FileSentAs.php

```diff
@@ -36,7 +36,15 @@ class FileSentAs extends ResponseBase
     public function matches($other): bool
     {
         /** @psalm-suppress PossiblyNullReference */
-        return $this->response->getFile()->getPathName() === $other;
+        $result = $this->response->getFile()->getPathName() === $other;
```

Comment:
This doesn't look right. Shouldn't you retrieve getFile() first and check it before calling getPathName()?